src/lib/config/common.js

```diff
@@ -8,18 +8,42 @@ export const DEFAULT_LANGUAGE = 'ENG';
 // text key is used for semantic ui dropdown options display
 
 export const ACQ_ORDER_STATUSES = [
-  { value: 'CANCELLED', text: 'Cancelled', label: 'Cancelled' },
-  { value: 'RECEIVED', text: 'Received', label: 'Received' },
-  { value: 'ORDERED', text: 'Ordered', label: 'Ordered' },
-  { value: 'PENDING', text: 'Pending', label: 'Pending' },
+  {
+    value: 'PENDING',
+    text: 'Pending',
+    label: 'Pending',
+    order: 1,
+    default: true,
+  },
+  {
+    value: 'ORDERED',
+    text: 'Ordered',
+    label: 'Ordered',
+    order: 2,
+    default: false,
+  },
+  {
+    value: 'RECEIVED',
+    text: 'Received',
+    label: 'Received',
+    order: 3,
+    default: false,
+  },
+  {
+    value: 'CANCELLED',
+    text: 'Cancelled',
+    label: 'Cancelled',
+    order: 4,
+    default: false,
+  },
 ];
 
 export const ILL_BORROWING_REQUESTS_STATUSES = [
-  { value: 'CANCELLED', text: 'Cancelled', label: 'Cancelled' },
-  { value: 'PENDING', text: 'Pending', label: 'Pending' },
-  { value: 'REQUESTED', text: 'Requested', label: 'Requested' },
-  { value: 'ON_LOAN', text: 'On loan', label: 'On loan' },
-  { value: 'RETURNED', text: 'Returned', label: 'Returned' },
+  { value: 'CANCELLED', text: 'Cancelled', label: 'Cancelled', order: 1 },
```

Comment:
Maybe it makes sense to add `order` and `default` to also **all** lists below and then change the code where they are used to take `order` and `default` into account?
WDYT? ping also @kprzerwa

Reply:
The code and configuration for using the order key was implemented in all lists of the common.js. Also, the code and configuration for using the default key was implemented in those lists that were not used in a form dedicated to update data (in other words, it was implemented for the first two lists).